src/components/CornerCircularLabel/CornerCircularLabel.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './CornerCircularLabel.scss';
+import {Corner} from '../../services/CommonTypes'
+
+interface Props {
```

Comment:
Component interfaces should be named for the component they refer to (e.g. `CornerCircularLabelProps`) for the ease of reference later on :)

Reply:
Does it make sense when the interfaces are strictly local? In the CornerCircularLabel I know that `Props` are `CornerCircularLabelProps`. Are there cases I would reference those props from outside the `CornerCircularLabel` file?